addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14100,7 +14100,29 @@ msgctxt "#36044"
 msgid "Disable polling for CEC capable devices?"
 msgstr ""
 
-#empty strings from id 36045 to 36100
+#. Desc of action to be taken when "changing CEC source" (1st)
```

Comment:
sorry for nitpicking, but these labels are no description but the actual value label of the actions, no? It also might be good to add a reference to the string ID of the according CEC setting.
Also, given that the setting is called `Action on switching...` shouldn't this value not just be `none`? (could be reused from existing strings)